### DISCUSSION TITLE
When playing large file audio, how to use streaming load, where to play and then load the corresponding number of bytes, rather than loading all the first time

### DISCUSSION THREAD

**Discussion by XIAOWEN-DEVS** [katspaugh/wavesurfer.js#3448] [Q&A]
When playing large file audio, how to use streaming load, where to play and then load the corresponding number of bytes, rather than loading all the first time

**Comment by katspaugh** [comment]
It's covered in the FAQ in the readme: https://github.com/katspaugh/wavesurfer.js#faq

**Reply by XIAOWEN-DEVS** [reply]
It didn't solve my problem. Your response was a bit perfunctor
